org.eclipse.xtext.sdk.feature/feature.xml

```diff
@@ -72,6 +72,8 @@ SPDX-License-Identifier: EPL-2.0
       <import plugin="org.eclipse.emf.ecore.source" version="2.26.0" match="greaterOrEqual"/>
       <import plugin="org.eclipse.emf.common.source" version="2.24.0" match="greaterOrEqual"/>
       <import feature="org.eclipse.emf.mwe2.language.sdk" version="2.19.0" match="greaterOrEqual"/>
+      <import plugin="org.apache.commons.commons-logging" version="1.3.4" match="greaterOrEqual"/>
```

Comment:
Should the required bundles be picked up from the manifests?

Reply:
Currently none of our bundles require that.